aries-site/src/data/structures/templates.js

```diff
@@ -275,4 +275,24 @@ export const templates = [
     sections: [],
     relatedContent: ['Page Layouts', 'Header', 'Navigation'],
   },
+  {
+    name: 'Pagination',
+    available: false,
+    description: `Pagination divides content into separate pages in order to 
+    enhance navigation to specific items.`,
+    preview: {
+      image: {
+        src: {
+          light: '/templateImages/template-preview-persistent.svg',
+          dark: '/templateImages/template-preview-persistent-invert.svg',
+        },
+        alt: 'HPE Design System Global Header',
+        fit: 'contain',
+      },
+    },
+    seoDescription: `Pagination divides content into separate pages in order to 
+    enhance navigation to specific items.`,
+    sections: [],
+    relatedContent: ['Table', 'Lists', 'Card'],
```

Comment:
Regardless of this PR, should we use a singular List instead of Lists to be consistent?

Reply:
That's a good point. I can file a clean up PR to use the component name instead of plural. Will we need to do a redirect for anyone who has potentially bookmarked the existing link?

Reply:
Good question, I think that if the page is not routing to 404 we can change it as is (e.g the home page), but if the Lists page would be routed to a 404 we might want to have a fallback if it's not too complicated.